tests/shap/test_log.py

```diff
@@ -40,19 +40,22 @@ def test_sklearn_log_explainer():
 
         run_id = run.info.run_id
 
-        X, y = shap.datasets.boston()
+        housing = fetch_california_housing(as_frame=True)
+        training_df = housing.data
+        target = housing.target
```

Comment:
```suggestion
        X, y = fetch_california_housing(as_frame=True, return_X_y=True)
```

Can we use `return_X_y`?

Reply:
The fetch-california-housing dataset contains 20640 rows. Can we sample them to run tests faster?

https://scikit-learn.org/stable/modules/generated/sklearn.datasets.fetch_california_housing.html

Reply:
yep! I'll change it real quick

Reply:
Updated!